src/devices/Pcd8544/Temperature.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Display.Pcd8544Enums
+{
+    /// <summary>
+    /// Temperature set for the PCD8544
+    /// </summary>
+    public enum Temperature
```

Comment:
Should this be internal? Also I'd suggest to change name of the enum because it will collide with UnitsNet

Reply:
I will rename it, it has to be public. It's actually called Temperature in the docs but will rename `ScreenTemperature`